README.md

```diff
@@ -46,6 +46,33 @@ sudo podman run \
     quay.io/centos-bootc/fedora-bootc:eln
 ```
 
+### Using local containers
+
+To use containers from local container's storage rather than a registry, we need to ensure two things:
+- the container exists in local storage
+- mount the local container storage
+
+Since the container is run in `rootful` only root container storage paths are allowed.
+
+```bash
+sudo podman run \
+    --rm \
+    -it \
+    --privileged \
+    --pull=newer \
+    --security-opt label=type:unconfined_t \
+    -v $(pwd)/config.json:/config.json \
+    -v $(pwd)/output:/output \
+    -v /var/lib/containers/storage:/var/lib/containers/storage \
```

Comment:
Hmm interesting, I'd suggested using `-v /var/lib/containers:/var/lib/containers` elsewhere...but perhaps this is indeed cleaner because AIUI it will avoid gathering e.g. created container roots (as opposed to the underlying images).